### ISSUE TITLE
Plugin installed, logged in, but videos don't play

### ISSUE THREAD

**Issue by sunfish62** [CastagnaIT/plugin.video.netflix#885]
## Bug report
I've finally gotten the netflix plugin installed under kodi 18.7, under a raspberry pi. Succeeded in getting logged in (this actually required running the authentication key on a separate machine and saving the key file-- but it worked). The netflix add on shows my user list, and each user's recently watched lists, which shows that the connection to Netflix is working. However, each time I try to run a video, the screen flashes for a second, and then... nothing. I've examined the kodi log, and see only a generic "cvideoplayer::openinputstream - error" associated with netflix. I'm at a loss. Help? 

#### Your Environment
- Netflix add-on version:  1.10.0
- Operating system version/name:  Raspberry OS Linux pi-4 5.4.51-v7l+
- Device model:  

Used Operating system:
* [ ] Android
* [ ] iOS
* [ ] Linux
* [ ] OSX
* [ ] Raspberry-Pi
* [ ] Windows

#### Steps to reproduce the behavior

1.
2.
3.

### Debug log

The debug log can be found from this link:

#### Installation
* [ ] I'm using other Netflix Repo 
* [ ] I'm using a different source

**Comment by Valanor** [comment]
same issue happen with 1.9.0_20200912 in kodi 18.7 raspi4

**Comment by Valanor** [comment]
> same issue happen with 1.9.0_20200912 in kodi 18.7 raspi4

just happen to be working again after reinstalling widevine

**Comment by sunfish62** [comment]
I will assume that your flagging this with "Ignored rules" relates to my failure to include a debug log, although it isn't clear from your response. 

To comply with your rules, please consult https://paste.kodi.tv/ocucuqokob

**Comment by CastagnaIT** [comment]
Whoever creates an Issue thread has the duty and responsibility to follow the instructions and not ignore them,
The instructions are in the Readme and also in the guide of the Issue thread creation.
OT Closed

> ERROR: AddOnLog: InputStream Adaptive: Unable to load widevine shared library (/home/pi/.kodi/cdm/libwidevinecdm.so)

This is not a error of NF add-on but a bug of InputStream Adaptive,
you have to follow these instructions:
https://github.com/emilsvennesson/script.module.inputstreamhelper/issues/395

**Comment by sunfish62** [comment]
Got it. Reinstalled widevine for a fourth time; this time it works.

**Comment by ashleysommer** [comment]
For anyone else coming here with this same problem, I just updated manually updated my Widevine using InputStream Helper.
The button is labelled "(Re)Install".
The widevine which doesn't work is "**4.10.1679.0 (13310.91.0)**"
I downgraded to 4.10.1610, and it gave me some weird other errors. Then I did the "(Re)Install" and it downloaded and installed version "**4.10.1679.0 (13310.93.0)**" <-- This one works!
